items_metadata.yaml

```diff
@@ -683,7 +683,76 @@ samples:
   licenseInfo: ""
   runtime: advanced_gpu
   tags: ["Data Science", "GIS", "Building", "Classification", "Deep Learning"]
-
+- title: Address Standardization and Correction using SequenceToSequence model
+  url: https://geosaurus.maps.arcgis.com/home/item.html?id=e8e2b0ac54584d079420b35b011d3d50
```

Comment:
This URL points to a same URL as "updating features in a feature layer", please change it to the correct URL